splunk_add_on_ucc_framework/commands/import_from_aob.py

```diff
@@ -36,3 +40,38 @@ def import_from_aob(addon_name: str) -> None:
         os.path.dirname(os.path.dirname(__file__)), "commands", "imports.py"
     )
     subprocess.call((import_from_aob_script_path, addon_name, imports_py_path))
+    addon_ucc_path = addon_name_directory + "_ucc"
+    a_path = os.path.join(*[os.getcwd(), addon_ucc_path, "package", "app.manifest"])
+    with open(a_path, encoding="utf-8") as f:
+        data = json.loads(f.read())
+
+    add_app_manifest_key(data, "supportedDeployments", "amsd")
+    add_app_manifest_key(data, "targetWorkloads", "amtw")
```

Comment:
You can simplify it by passing `ap.APP_MANIFEST_SUPPORTED_DEPLOYMENTS` or `ap.APP_MANIFEST_TARGET_WORKLOADS` directly.